Automated testing in WordPress/Codeception.md

```diff
@@ -2,18 +2,319 @@ Codeception collects and shares recommended practices and solutions for testing
 
 ## Codeception for WordPress
 
-You can find full installation instructions [here](https://codeception.com/for/wordpress).
+You can find full installation instructions [here](https://codeception.com/for/wordpress) or in the [official documentation](https://wpbrowser.wptestkit.dev/) for wp-browser extension.
 
-First, install the latest stable WPBrowser package via Composer.
+### Installation
+
+First, install the latest stable `WPBrowser` package via Composer in the root folder of your project. If your project is a plugin then it should be installed in the root folder of your plugin; if your project is a theme it should be installed in the root folder of your theme.
 
 ```bash
-composer require lucatume/wp-browser --dev
+composer require --dev lucatume/wp-browser
 ```
 
-Then, while in the root of your plugin or theme where you've installed the codeception, run
+In order for your tests to work, you'll need to set up a test instance. You can your preferred method of development environments - Valet+, VVV, Docker, Local by Flywheel etc. It's important to have a working WordPress instance, so that you can run acceptance and integration tests on it.
```

Comment:
```suggestion
In order for your tests to work, you'll need to set up a test instance. You can use your preferred method of setting up development environments - Valet+, VVV, Docker, Local by Flywheel etc. It's important to have a working WordPress instance, so that you can run acceptance and integration tests on it.
```